apps/server/src/stores/runtimeState.ts

```diff
@@ -613,11 +605,13 @@ export function roll(
   runtimeState.runtime.offset = offset;
   const offsetClock = runtimeState.clock + runtimeState.runtime.offset;
 
+  //TODO: improve this, for now just made to work
+  const timedEvents = metadata.timedEventOrder.map((id) => rundown.entries[id]) as OntimeEvent[];
```

Comment:
question: what improvements do you suggest?
if we push this forward it is likely that we forget where is the room for improvement so we are just left with an ominous comment...